docs/modules/ROOT/pages/object-storage/create.adoc

```diff
@@ -45,6 +45,8 @@ If you see the "True" value in the READY column, it means that your bucket is pr
 +
 IMPORTANT: If your object bucket doesn't get the state ready after a while, try to delete the bucket and re-create it with a different name.
 
+PROBABLY JUST LINK TO THE SECRETS PAGE HERE?
```

Comment:
Please mark these texts with the "TODO:" prefix in the future, to make them easier to spot in the text

Reply:
Will keep in mind, thanks! 👍